test/Groups/group_characters.jl

```diff
@@ -21,12 +21,12 @@
   @test String(take!(io)) == "Alt( [ 1 .. 4 ] )\n\n 2  2  2  .  .\n 3  1  .  1  1\n              \n   1a 2a 3a 3b\n2P 1a 1a 3b 3a\n3P 1a 2a 1a 1a\n              \nχ₁  1  1  1  1\nχ₂  1  1  A  A̅\nχ₃  1  1  A̅  A\nχ₄  3 -1  .  .\n\nA = -ζ₃ - 1\nA̅ = ζ₃\n"
 
   # ... and in LaTeX format
-  show(IOContext(io, :with_legend => true), MIME("text/html"), t_a4)
+  show(IOContext(io, :with_legend => true), MIME("text/latex"), t_a4)
   @test String(take!(io)) == "\$Alt( [ 1 .. 4 ] )\n\n\\begin{array}{rrrrr}\n2 & 2 & 2 & . & . \\\\\n3 & 1 & . & 1 & 1 \\\\\n &  &  &  &  \\\\\n & 1a & 2a & 3a & 3b \\\\\n2P & 1a & 1a & 3b & 3a \\\\\n3P & 1a & 2a & 1a & 1a \\\\\n &  &  &  &  \\\\\n\\chi_{1} & 1 & 1 & 1 & 1 \\\\\n\\chi_{2} & 1 & 1 & A & \\overline{A} \\\\\n\\chi_{3} & 1 & 1 & \\overline{A} & A \\\\\n\\chi_{4} & 3 & -1 & . & . \\\\\n\\end{array}\n\nA = -\\zeta_{3} - 1\n\\overline{A} = \\zeta_{3}\n\$"
 
   # show the screen format for a table with real and non-real irrationalities
   show(IOContext(io, :with_legend => true), character_table("L2(11)"))
-  @test String(take!(io)) == "L2(11)\n\n  2  2  2  1  .  .  1   .   .\n  3  1  1  1  .  .  1   .   .\n  5  1  .  .  1  1  .   .   .\n 11  1  .  .  .  .  .   1   1\n                             \n    1a 2a 3a 5a 5b 6a 11a 11b\n 2P 1a 1a 3a 5b 5a 3a 11b 11a\n 3P 1a 2a 1a 5b 5a 2a 11a 11b\n 5P 1a 2a 3a 1a 1a 6a 11a 11b\n11P 1a 2a 3a 5a 5b 6a  1a  1a\n                             \n χ₁  1  1  1  1  1  1   1   1\n χ₂  5  1 -1  .  .  1   C   C̅\n χ₃  5  1 -1  .  .  1   C̅   C\n χ₄ 10 -2  1  .  .  1  -1  -1\n χ₅ 10  2  1  .  . -1  -1  -1\n χ₆ 11 -1 -1  1  1 -1   .   .\n χ₇ 12  .  .  A  B  .   1   1\n χ₈ 12  .  .  B  A  .   1   1\n\nA = -ζ₅³ - ζ₅² - 1\nB = ζ₅³ + ζ₅²\nC = ζ₁₁⁹ + ζ₁₁⁵ + ζ₁₁⁴ + ζ₁₁³ + ζ₁₁\nC̅ = -ζ₁₁⁹ - ζ₁₁⁵ - ζ₁₁⁴ - ζ₁₁³ - ζ₁₁ - 1\n"
+  @test String(take!(io)) == "L2(11)\n\n  2  2  2  1  .  .  1   .   .\n  3  1  1  1  .  .  1   .   .\n  5  1  .  .  1  1  .   .   .\n 11  1  .  .  .  .  .   1   1\n                             \n    1a 2a 3a 5a 5b 6a 11a 11b\n 2P 1a 1a 3a 5b 5a 3a 11b 11a\n 3P 1a 2a 1a 5b 5a 2a 11a 11b\n 5P 1a 2a 3a 1a 1a 6a 11a 11b\n11P 1a 2a 3a 5a 5b 6a  1a  1a\n                             \n χ₁  1  1  1  1  1  1   1   1\n χ₂  5  1 -1  .  .  1   B   B̅\n χ₃  5  1 -1  .  .  1   B̅   B\n χ₄ 10 -2  1  .  .  1  -1  -1\n χ₅ 10  2  1  .  . -1  -1  -1\n χ₆ 11 -1 -1  1  1 -1   .   .\n χ₇ 12  .  .  A A*  .   1   1\n χ₈ 12  .  . A*  A  .   1   1\n\nA = -ζ₅³ - ζ₅² - 1\nA* = ζ₅³ + ζ₅²\nB = ζ₁₁⁹ + ζ₁₁⁵ + ζ₁₁⁴ + ζ₁₁³ + ζ₁₁\nB̅ = -ζ₁₁⁹ - ζ₁₁⁵ - ζ₁₁⁴ - ζ₁₁³ - ζ₁₁ - 1\n"
```

Comment:
Can we perhaps at least format these as triple quoted strings, that'd be slightly nicer for a human to read? I.e. like this (untested)
```suggestion
  @test String(take!(io)) == 
    """
    L2(11)

      2  2  2  1  .  .  1   .   .
      3  1  1  1  .  .  1   .   .
      5  1  .  .  1  1  .   .   .
     11  1  .  .  .  .  .   1   1
                         
        1a 2a 3a 5a 5b 6a 11a 11b
     2P 1a 1a 3a 5b 5a 3a 11b 11a
     3P 1a 2a 1a 5b 5a 2a 11a 11b
     5P 1a 2a 3a 1a 1a 6a 11a 11b
    11P 1a 2a 3a 5a 5b 6a  1a  1a
                         
     χ₁  1  1  1  1  1  1   1   1
     χ₂  5  1 -1  .  .  1   B   B̅
     χ₃  5  1 -1  .  .  1   B̅   B
     χ₄ 10 -2  1  .  .  1  -1  -1
     χ₅ 10  2  1  .  . -1  -1  -1
     χ₆ 11 -1 -1  1  1 -1   .   .
     χ₇ 12  .  .  A A*  .   1   1
     χ₈ 12  .  . A*  A  .   1   1

    A = -ζ₅³ - ζ₅² - 1
    A* = ζ₅³ + ζ₅²
    B = ζ₁₁⁹ + ζ₁₁⁵ + ζ₁₁⁴ + ζ₁₁³ + ζ₁₁
    B̅ = -ζ₁₁⁹ - ζ₁₁⁵ - ζ₁₁⁴ - ζ₁₁³ - ζ₁₁ - 1
    """
```